workitems/l2-transaction-fees/l2-transaction-fees-v1.0-psd01.md

```diff
@@ -0,0 +1,334 @@
+- Identify and document characteristics of Layer 2 Blockchain environments for EVM compatible public blockchains that will be key in addressing mainstream and enterprise adoption.
+
+The work is an [EEA Community Project](https://entethalliance.org/eeacommunityprojects/), which is managed by [OASIS](https://oasis-open-projects.org/).
+
+## 1.1 Overview
+
+... 
+
+
+## 1.2 Glossary
+
+**Blockchain:**
+
+An open, distributed ledger that can record transactions between two parties efficiently and in a verifiable and permanent way.
+
+**Gas**
+
+Tefers to the unit that measures the amount of computational and storage effort required to execute specific operations on an EVM-compatible network.
+
+**Base Fee**
+
+The Layer 2 gas price. 
+
+Note that a gas price is typically variable and changes with the level of usage of a Layer 2 network and is expressed in Giga Wei (GWEI) for EVM-compatible networks, where Wei represents the smallest unit of gas; 1 Wei = 10<sup>-18</sup> Eth in Ethereum's native token. Also note, that typically there is a minimal, fixed base fee which normally varies by Layer 2 network. 
+
+**Execution Fee**
+
+A value when multiplied by a base fee will be sufficient to cover both the Layer 2 and Layer 1 transaction fees.
+
+An example of such a calculation is:
+$$L2\space Gas\space Limit + {L1\space Transaction\space Fee \over L2\space Gas\space Price}$$
+
+**Layer 1:**
+
+A base network, such as Bitcoin, or Ethereum, and its underlying infrastructure that validates and finalizes transactions without the need of another network.
+
+**Layer 2:**
+
+A secondary framework or protocol that is built on top of an existing Layer 1 system in such a way that it inherits the security properties of the Layer 1 system while allowing for a higher transaction throughput that the Layer 1 system.
+
+**Maximal Extractable Value**
+
+Refers to the maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.
+
+**Priorirty Fee**
+
+To be paid by the transaction originator to a Layer 2 sequencer to obtain a desired slot for its transaction in a new block.
+
+Note that the exact position of a transaction in a new block may be determined by factors such as time-stamp, or minimization of Maximal Extractable Value (MEV) of a block in addition to a Priority Fee.
+
+**Sequencer**
+
+Collects transactions, publishes them in a batch to the Layer 1 on which the Layer 2 operates, receives transaction fees from the published transactions, pays Layer 2 fees to other Layer 2 protocol participants, and, if required, participates in a consensus algorithm with other sequencers to determine transaction  ordering in a block.
+
+
+**Sidechain:**
+
+A secondary blockchain connected to the main blockchain with a two-way peg.
```

Comment:
This term is sometimes used to refer specifically to constructions that introduce distinct security assumptions (i.e., BFT sidechains) and sometimes as a super-category which includes the above along with L2s like rollups; this definition seems to be the later (not sure what we want to go with / what's best)

Reply:
@DZGoldman what is your recommendation?

Reply:
will add `and have its own set of trust assumptions`

Reply:
per WG 1/11/23